analytics/migrations/0009_remove_messages_to_stream_stat.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db import migrations
+
+from analytics.lib.counts import do_delete_count_stat
+
+def delete_messages_sent_to_stream_stat(apps, schema_editor):
+    # type: (StateApps, DatabaseSchemaEditor) -> None
+    do_delete_count_stat('messages_sent_to_stream:is_bot')
```

Comment:
(It's kinda an interesting issue, since we don't want this to hang if an analytics cron job is running; I guess I could imagine trying to kill the analytics job).

Might make sense to just not deal with the race condition for now; it's likely pretty minor and I think there may be more risk of problems in trying to deal with it.